lms/djangoapps/ccx/tests/test_views.py

```diff
@@ -615,6 +619,30 @@ def test_student_progress(self):
         self.assertEqual(len(grades['section_breakdown']), 4)
 
 
+@attr('shard_1')
+@override_settings(FIELD_OVERRIDE_PROVIDERS=('ccx.overrides.CustomCoursesForEdxOverrideProvider',))
+@patch('xmodule.x_module.XModuleMixin.get_children', patched_get_children, spec=True)
+@override_settings(FEATURES=FEATURES_WITH_CUSTOM_GRADING, GRADING_TYPE='vertical')
+class TestCustomGradeVerticals(CCXGradesMixin, ModuleStoreTestCase, LoginEnrollmentTestCase):
```

Comment:
Why not use 3 arguments to override_settings, but 2 calls of it? It requires this exact order?